database/base_db.py

```diff
@@ -1,57 +1,43 @@
 from motor.motor_asyncio import AsyncIOMotorClient, AsyncIOMotorDatabase, AsyncIOMotorCollection
 from config.env import MONGO_URI, MONGO_DB_NAME
 import logging
-from pymongo import MongoClient
-import pymongo.collection
-import pymongo.database
-from typing import Optional, Union, List
+from typing import Optional, List
 from utilities.types import MongoMappingType, JSONDict
 
 
 class BaseDB:
 
-    async_client: Optional[AsyncIOMotorClient] = None  # type: ignore[no-any-unimported]
-    client: Optional[MongoClient[MongoMappingType]] = None
+    client: Optional[AsyncIOMotorClient] = None  # type: ignore[no-any-unimported]
 
     @classmethod
-    def _get_client_async(cls) -> AsyncIOMotorClient:  # type: ignore[no-any-unimported]
-        if not cls.async_client:
-            cls.async_client = AsyncIOMotorClient(MONGO_URI)
-            cls._verify_connection(cls.async_client)
-        return cls.async_client
-
-    @classmethod
-    def _get_client(cls) -> MongoClient[MongoMappingType]:
+    def _get_client(cls) -> AsyncIOMotorClient:  # type: ignore[no-any-unimported]
         if not cls.client:
-            cls.client = MongoClient(MONGO_URI)
+            cls.client = AsyncIOMotorClient(MONGO_URI)
             cls._verify_connection(cls.client)
         return cls.client
 
     @classmethod
-    def _verify_connection(cls, client: Union[MongoClient[MongoMappingType], AsyncIOMotorClient]) -> None:  # type: ignore[no-any-unimported]
+    def _verify_connection(cls, client: AsyncIOMotorClient) -> None:  # type: ignore[no-any-unimported]
         try:
             client.server_info()
         except Exception as e:
             logging.critical("Cannot connect to db.")
             raise e
 
     @classmethod
-    def get_database_async(cls) -> AsyncIOMotorDatabase:  # type: ignore[no-any-unimported]
-        return cls._get_client_async()[MONGO_DB_NAME]
-
-    @classmethod
-    def get_database(cls) -> pymongo.database.Database[MongoMappingType]:
+    def get_database(cls) -> AsyncIOMotorDatabase:  # type: ignore[no-any-unimported]
         return cls._get_client()[MONGO_DB_NAME]
 
+    # TODO this needs to be reworked
```

Comment:
This will be done after this gets merged https://github.com/castlepointanime/backend/pull/41

Once that gets merged I'll make an issue to fix the auto approver assigning system